UM/AppVersionFile.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2019 Ultimaker B.V.
+# Uranium is released under the terms of the LGPLv3 or higher.
+from configparser import ConfigParser
+
+from UM.Logger import Logger
+from UM.Version import Version
+
+
+class AppVersionFile:
+
+    #
+    # Constructor. The app_name and version provided here should be the information of the current application.
+    #
+    def __init__(self, app_name: str, version: str) -> None:
+        self.app_name = app_name
+        self.version = version
```

Comment:
I guess it's better to use the version object here right away (otherwise we can get confusion about in which cases we use str or Version)